core/trino-main/src/test/java/io/trino/sql/analyzer/TestAnalyzer.java

```diff
@@ -7430,6 +7432,191 @@ public void testDisallowAggregationFunctionInUnnest()
                 .hasMessage("line 1:46: UNNEST cannot contain aggregations, window functions or grouping operations: [COUNT(t.a)]");
     }
 
+    @Test
+    public void testSelectColumnLineageInfo()
+    {
+        String sql = "SELECT a, b + 1 AS b1, 'C' as literal, a + b FROM (VALUES (1, 2)) t(a, b)";
+
+        Analysis analysis = analyze(sql);
+
+        Optional<List<ColumnLineageInfo>> maybeLineageInfo = analysis.getSelectColumnLineageInfo();
```

Comment:
Rename `maybeLineageInfo` to `optionalLineageInfo`. Here and in the other test cases.